queue_job/jobrunner/runner.py

```diff
@@ -181,15 +180,8 @@ def _channels():
     )
 
 
-def _datetime_to_epoch(dt):
-    # important: this must return the same as postgresql
-    # EXTRACT(EPOCH FROM TIMESTAMP dt)
-    return (dt - datetime.datetime(1970, 1, 1)).total_seconds()
-
-
 def _odoo_now():
-    dt = datetime.datetime.utcnow()
-    return _datetime_to_epoch(dt)
+    return time.time()
```

Comment:
I'd keep the comment that says it must do the same as the postgres equivalent, though.

Reply:
Indeed, done :+1: